src/toil/test/wdl/toilwdlTest.py

```diff
@@ -1,522 +0,0 @@
-        assert parse_disk('local-disk 10 SSD') == 10000000000, str(parse_disk('local-disk 10 SSD'))
-        assert parse_disk('/mnt/ 10 HDD') == 10000000000, str(parse_disk('/mnt/ 10 HDD'))
-        assert parse_disk('/mnt/ 1000 HDD') == 1000000000000, str(parse_disk('/mnt/ 1000 HDD'))
-
-    # estimated run time <1 sec
-    def testPrimitives(self):
-        """Test if toilwdl correctly interprets some basic declarations."""
-        wdl = os.path.abspath('src/toil/test/wdl/testfiles/vocab.wdl')
-
-        # TODO: test for all version.
-        aWDL = get_analyzer(wdl)
-        aWDL.analyze()
-
-        no_declaration = ['bool1', 'int1', 'float1', 'file1', 'string1']
-        collection_counter = []
-        for key, declaration in aWDL.workflows_dictionary['vocabulary'].items():
-            if not key.startswith('declaration'):
-                continue
-
-            name, var_type, var_expr = declaration
-
-            if name in no_declaration:
-                collection_counter.append(name)
-                assert not var_expr
-
-            if name == 'bool2':
-                collection_counter.append(name)
-                assert var_expr == 'True', var_expr
-                assert var_type == 'Boolean', var_type
-            if name == 'int2':
-                collection_counter.append(name)
-                assert var_expr == '1', var_expr
-                assert var_type == 'Int', var_type
-            if name == 'float2':
-                collection_counter.append(name)
-                assert var_expr == '1.1', var_expr
-                assert var_type == 'Float', var_type
-            if name == 'file2':
-                collection_counter.append(name)
-                assert var_expr == "'src/toil/test/wdl/test.tsv'", var_expr
-                assert var_type == 'File', var_type
-            if name == 'string2':
-                collection_counter.append(name)
-                assert var_expr == "'x'", var_expr
-                assert var_type == 'String', var_type
-        assert collection_counter == ['bool1', 'int1', 'float1', 'file1', 'string1',
-                                      'bool2', 'int2', 'float2', 'file2', 'string2']
-                                      
-    # estimated run time <1 sec
-    def testCSV(self):
-        default_csv_output = [['1', '2', '3'],
-                              ['4', '5', '6'],
-                              ['7', '8', '9']]
-        csv_array = read_csv(os.path.abspath('src/toil/test/wdl/test.csv'))
-        assert csv_array == default_csv_output
-
-    # estimated run time <1 sec
-    def testTSV(self):
-        default_tsv_output = [['1', '2', '3'],
-                              ['4', '5', '6'],
-                              ['7', '8', '9']]
-        tsv_array = read_tsv(os.path.abspath('src/toil/test/wdl/test.tsv'))
-        assert tsv_array == default_tsv_output
-
-class ToilWdlIntegrationTest(BaseToilWdlTest):
```

Comment:
We might want to port some of these tests over to the interpreter at some point; the conformance tests cover conformance with individual spec elements and don't really cover integration with popular WDL workflows.